src/js/main.js

```diff
@@ -0,0 +1,31 @@
+jQuery(document).ready(function($) {
+  $('.burger-btn').click(function(event) {
+    $('.sidebar').toggleClass('active');
+    $('.navigation').toggleClass('active');
+    $('.content').toggleClass('active');
+  });
+
+  $('.mobile-nav-list-item').click(function(event) {
+    localStorage.setItem('clicked-link', event.target.id);
```

Comment:
Local storage is redundant here. You can just set appropriate class, setActiveLink function does it

Reply:
I don't understand, how i can get id of clicked element of after page reload. I think that locale storage is the best way to get it. Maybe i am wrong. Do you any better ideas?